packages/cli/src/services/util.ts

```
@@ -73,28 +67,63 @@ export async function loadJsFile (filepath: string): Promise<any> {
+  // Backward-compatibility for users who installed ts-node.
+  const tsCompiler = await getTsNodeService()
+  if (tsCompiler) {
     tsCompiler.enabled(true)
-    let { default: exported } = await require(filepath)
-    if (exported instanceof Function) {
-      exported = await exported()
+    let exported: any
+    try {
+      exported = (await require(filepath)).default
+    } catch (err: any) {
+      if (err.message && err.message.contains('Unable to compile TypeScript')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)